### PR TITLE
navbar and some style

### PR DESCRIPTION
**Pull request by memeraki**

### CODE REVIEW

**Review by AleksandraCyp — COMMENTED** [review]
It all looks great in desktop! The only thing to change is adding extra routs in App.js - instead of the route /me we should have exact paths /me/books me/pokes me/baskets ect. 
Plus please add log-out option

File: src/App.js [hunk 1]
```diff
@@ -89,6 +95,46 @@ function App() {
...
           ></Route>
+          <Route
+            path="/me/books"
+            render={() =>
+              accessToken ? (
+                <MeBooks accessToken={accessToken} />
+              ) : (
+                <Redirect to="/" />
+              )
+            }
```
Comment (lukaszdutka) [inline src/App.js:106]: I think it can be collapsed to one line, but it's preference

**Review by lukaszdutka — COMMENTED** [review]
In terms of code - Looks good to me. However, I am not aware of those functionalities - so maybe some more eyes should look at it 👍

**Review by AleksandraCyp — APPROVED** [review]
It looks and works great, I love the mobile version :)

**Pull request merged**